src/main/java/com/harrison/config/EmbeddedPostgresConfig.java

```diff
@@ -3,6 +3,7 @@
 import java.io.IOException;
 
 import javax.sql.DataSource;
+import javax.sql.DataSource;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this duplicated import. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-54-187-4-185.us-west-2.compute.amazonaws.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)